packages/postgres-database/src/simcore_postgres_database/utils_products.py

```diff
@@ -0,0 +1,32 @@
+""" Common functions to access products table
+
+"""
+
+from typing import Protocol
+
+import sqlalchemy as sa
+
+from .models.products import products
+
+
+class _DBConnection(Protocol):
+    # Prototype to account for aiopg and asyncio connection classes, i.e.
+    #   from aiopg.sa.connection import SAConnection
+    #   from sqlalchemy.ext.asyncio import AsyncConnection
+    async def scalar(self, *args, **kwargs):
+        ...
+
+
+async def get_default_product_name(conn: _DBConnection) -> str:
+    """The first row in the table is considered as the default product
+
+    :: raises ValueError if undefined
+    """
+    product_name = await conn.scalar(
```

Comment:
so the obvious question.. who creates the entry in the database?

Reply:
It has to be done by an operator

Reply:
But if it is not done, the services do not start